### DISCUSSION TITLE
Specify the app's homepage for the installers, esp. MSI

### DISCUSSION THREAD

**Discussion by sschuberth** [tauri-apps/tauri#9943] [Q&A] [answered]
Hi there, is it currently possible to specify an app's homepage as part of installer metadata? Looking at https://tauri.app/v1/api/config/#windowsconfig and https://tauri.app/v1/api/config/#wixconfig it does not seem it's possible. I'm specifically looking for doing that for MSI, so that the app's homepage shows up in Windows' uninstall dialog.

**Comment by FabianLars** [comment, marked as answer]
> Hi there, is it currently possible to specify an app's homepage as part of installer metadata?

Only by modifying the wix (for msi) and nsis (for setup.exe) templates https://tauri.app/v1/guides/building/windows#customizing-the-wix-installer-template

Feel free to open a feature request (via a github issue) for this though :)
